developer/src/common/web/utils/src/xml-utils.ts

```diff
@@ -24,60 +24,81 @@ type KemanXMLOptionsBag = {
 /** map of options for the XML parser */
 const PARSER_OPTIONS: KemanXMLOptionsBag = {
```

Comment:
```suggestion
const PARSER_OPTIONS: KeymanXMLOptionsBag = {
```

I should have picked this up before. But let's fix before it stagnates :grin: